lib/plugins/entities.js

```diff
@@ -31,6 +31,11 @@ function inject (bot, { version }) {
   const entitiesArray = require('minecraft-data')(version).entitiesArray
   const Item = require('prismarine-item')(version)
   const ChatMessage = require('prismarine-chat')(version)
+  // ONLY 1.17 has this destroy_entity packet which is the same thing as entity_destroy packet except the entity is singular
+  // 1.17.1 reverted this change so this is just a simpler fix
+  bot._client.on('destroy_entity', (packet) => {
+    bot._client.emit('entity_destroy', { entityIds: [packet.entityId] })
```

Comment:
it's a hack to emit packet events like this, it will prevent things like mineflayer proxy from working well
let's use an event on bot instead ?